isaric/parsers/ccp-cameroon.toml

```diff
@@ -415,10 +406,10 @@
     7 = "palliative discharge"
 
 [visit.date_outcome]
-  combinedType = "min"
+  combinedType = "firstNonNull"
   fields = [
-    { field = "q9_outcome" },
     { field = "q14_treatment_outcome", description = "14) End date of treatment / Date d'arrêt du traitement" },
+    { field = "q9_outcome" }
```

Comment:
@ekamau Here I think we need to be careful to match up the outcome dates with the outcome fields, perhaps by adding conditions for which outcome field is populated. For example there are patients with a treatment outcome given in q1_treatment_outcome but no treatment outcome date given in q14_treatment_outcome. I think using the above code we could get the outcome from treatment but the date from q9_outcome (initial consultation outcome date), which would be incorrect.

Reply:
@ekamau let me know if this needs more discussion or is now resolved - thanks!

Reply:
@pipliggins - could you also help me check here if parser now reflects Sadie's comment? should we add 'q1_treatment_outcome' or not? thank you :)